mscore/plugin/api/part.h

```diff
@@ -25,10 +25,32 @@ namespace PluginAPI {
 
 class Part : public Ms::PluginAPI::ScoreElement {
       Q_OBJECT
-      Q_PROPERTY(int                            startTrack          READ startTrack)
-      Q_PROPERTY(int                            endTrack            READ endTrack)
+      Q_PROPERTY(int                            startTrack           READ startTrack)
+      Q_PROPERTY(int                            endTrack             READ endTrack)
       /// The string identifier for the current instrument. \since MuseScore 3.2
-      Q_PROPERTY(QString                        instrumentId        READ instrumentId)
+      Q_PROPERTY(QString                        instrumentId         READ instrumentId)
+      /// TBD \since MuseScore 3.2
+      Q_PROPERTY(int                            harmonyCount         READ harmonyCount)
+      /// TBD \since MuseScore 3.2
+      Q_PROPERTY(bool                           hasDrumStaff         READ hasDrumStaff)
+      /// TBD \since MuseScore 3.2
+      Q_PROPERTY(bool                           hasPitchedStaff      READ hasPitchedStaff)
+      /// TBD \since MuseScore 3.2
+      Q_PROPERTY(bool                           hasTabStaff          READ hasTabStaff)
+      /// TBD \since MuseScore 3.2
+      Q_PROPERTY(int                            lyricCount           READ lyricCount)
+      /// TBD \since MuseScore 3.2
+      Q_PROPERTY(int                            midiChannel          READ midiChannel)
+      /// TBD \since MuseScore 3.2
+      Q_PROPERTY(int                            midiProgram          READ midiProgram)
+      /// TBD \since MuseScore 3.2
+      Q_PROPERTY(QString                        longName             READ longName        WRITE setLongName)
+      /// TBD \since MuseScore 3.2
+      Q_PROPERTY(QString                        shortName            READ shortName       WRITE setShortName)
+      /// TBD \since MuseScore 3.2
+      Q_PROPERTY(QString                        partName             READ partName        WRITE setPartName)
+      /// TBD \since MuseScore 3.2
```

Comment:
whether part is shown or hidded, resp. whether to show or hide it.
`showPart()` and `setShotPart()`might be better names here

Reply:
>showPart() and setShowPart() might be better names here

That would be a change in implementation from the QML level.

Reply:
So what? We did that in other places too, like poet() vs lyricist()

Reply:
may get added to the plugin converter then, but is possible and now the right time to do.

Reply:
>So what? We did that in other places too, like poet() vs lyricist()
>may get added to the plugin converter then, but is possible and now the right time to do.

I don't disagree with any of this.

This commit is just not the place to debate it. I'm not architecturally qualified to weigh in on its merits. I'm just a bit contributor trying to fix something I ran up against for an existing plugin. Just digging some ditches as it were.

My little bit work was simply porting existing behaviors.

-Dale